easybuild/toolchains/craymkl.py

```diff
@@ -0,0 +1,93 @@
+##
+# Copyright 2014-2019 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+Support for Cray/Intel MKL library, which provides BLAS/LAPACK support.
+
+:author: Petar Forai (IMP/IMBA, Austria)
+:author: Kenneth Hoste (Ghent University)
+:author: Guilherme Peretti-Pezzi (CSCS)
+"""
+import os
+
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.toolchain.linalg import LinAlg
+
+
+CRAY_MKL_MODULE_NAME = 'intel'
```

Comment:
No idea what this is about, opened an issue: https://github.com/houndci/hound/issues/1769

Reply:
Black would change this to `"intel"` is I think what is meant here.

Reply:
Yeah, that could be it... 🤦‍♂ 

I'll look into disabling `flake8-black` though (as suggested in https://github.com/houndci/hound/issues/1769), since unless you're actually using the `black` code formatter (maybe we should, but it's a bit of effort to start using it & enforcing it), that's going to be a nightmare...

Reply:
disabled via #3162